src/test/java/uk/gov/companieshouse/chd/order/api/service/OrderServiceTest.java

```diff
@@ -49,4 +70,77 @@ void saveOrderDetailsPersistsOrderHeader(){
         // when and then
         assertThat(serviceUnderTest.saveOrderDetails(midRequest), is(orderHeader));
     }
+
+    @Test
+    void paymentRefHasBarcodeAppended() {
+        final MissingImageDeliveriesRequest midRequest = new MissingImageDeliveriesRequest();
+        midRequest.setId(ID);
+        midRequest.setCompanyNumber(COMPANY_NUMBER);
+        midRequest.setCompanyName(COMPANY_NAME);
+        midRequest.setPaymentReference(PAYMENT_REFERENCE);
+        midRequest.setOrderedAt(ORDERED_AT);
+        midRequest.setItemCost(ITEM_COST);
+        midRequest.setFilingHistoryCategory(FILING_HISTORY_CATEGORY);
+        midRequest.setFilingHistoryDate(FILING_HISTORY_DATE);
+        midRequest.setFilingHistoryDescription(FILING_HISTORY_DESCRIPTION);
+        midRequest.setFilingHistoryType(FILING_HISTORY_TYPE);
+        midRequest.setFilingHistoryBarcode(FILING_HISTORY_BARCODE);
```

Comment:
This code block appears to be duplicated. We should extract this to a method or set up the object 1 time and remove whatever is required for the test instead of duplicating like this.